apps/passport-server/src/database/models.ts

```diff
@@ -330,3 +330,10 @@ export interface AnonMessageWithDetails extends AnonMessage {
   topic_name: string;
   reactions: string[];
 }
+
+export interface RateLimit {
+  type: "CHECK_EMAIL_TOKEN" | "REQUEST_EMAIL_TOKEN";
+  name: string;
+  count: number;
+  expiry: number;
+}
```

Comment:
nit: to me, this interface does not map cleanly to either of the tables you created above. I am slightly confused by what `name` refers to here, whether `count` is the remaining or used up limit, and what `expiry` means.